browser-test/src/admin/admin_application_statuses.test.ts

```diff
@@ -359,6 +361,171 @@ test.describe('view program statuses', () => {
 
       expect(await adminPrograms.getNoteContent()).toBe(noteText)
     })
+
+    test.describe('email status updates work correctly with PAI flag on', () => {
+      test.beforeAll(async () => {
+        const {page, adminPrograms, adminQuestions} = ctx
+        await enableFeatureFlag(
+          page,
+          'primary_applicant_info_questions_enabled',
+        )
+        await loginAsAdmin(page)
+        await adminQuestions.addEmailQuestion({
+          questionName: 'Email',
+          universal: true,
+          primaryApplicantInfo: true,
+        })
+        await adminPrograms.editProgram(programWithStatusesName)
+        await adminPrograms.editProgramBlock(
+          programWithStatusesName,
+          'block description',
+          ['Email'],
+        )
+        await adminPrograms.publishAllDrafts()
+        await logout(page)
+      })
```

Comment:
Using `beforeAll` and `afterAll` creates a shared state between the tests which can lead to tests relying on other tests.

Reply:
That makes sense and explains why I was seeing some bleed over. Would you recommend I move this logic to `beforeEach`?

Reply:
Yeah, I would let the `beforeEach` handle it. One thing to keep in mind is that way up at the top your ctx is created with clearDb=false so you'll probably have to call the method to clear the database yourself.

Reply:
Done.